src/Symfony/Component/Webhook/Server/Transport.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Webhook\Server;
+
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\HttpClient\HttpOptions;
+use Symfony\Component\RemoteEvent\Event;
+use Symfony\Component\Webhook\Subscriber;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 6.3
+ */
+class Transport implements TransportInterface
+{
+    public function __construct(
+        private HttpClientInterface $client,
```

Comment:
can we use `readonly` everywhere it applies?

Reply:
Done